api-server/src/common/models/user.json

```diff
@@ -208,6 +208,7 @@
           "solution": "string",
           "githubLink": "string",
           "challengeType": "number",
+          "manuallyApproved": "boolean",
```

Comment:
Can I be a stickler and request this be `isManuallyApproved`?

---

Do not change it, on account of me, let us see what others think too. I just like booleans having that convention.

Reply:
I like it too, it's a good convention.